src/inspector_socket.cc

```diff
@@ -108,12 +108,12 @@ static int write_to_client(inspector_socket_t* inspector,
 #endif
 
   // Freed in write_request_cleanup
-  uv_buf_t* buf = reinterpret_cast<uv_buf_t*>(malloc(sizeof(uv_buf_t)));
-  uv_write_t* req = reinterpret_cast<uv_write_t*>(malloc(sizeof(uv_write_t)));
+  uv_buf_t* buf = reinterpret_cast<uv_buf_t*>(NODE_MALLOC(sizeof(uv_buf_t)));
+  uv_write_t* req = reinterpret_cast<uv_write_t*>(NODE_MALLOC(sizeof(uv_write_t)));
```

Comment:
This line is > 80 chars, not too sure what a *clean* way to break it would be. Any suggestions?
Same with line 478.